src/components/settings/SettingsAccessTab/SettingsAccessLinkManagement.jsx

```diff
@@ -178,12 +188,20 @@ const SettingsAccessLinkManagement = ({
 const mapStateToProps = (state) => ({
   enterpriseUUID: state.portalConfiguration.enterpriseId,
   isUniversalLinkEnabled: state.portalConfiguration.enableUniversalLink,
+  couponsData: state.coupons.data ? camelCaseObject(state.coupons.data) : { results: [] },
+});
+
+const mapDispatchToProps = dispatch => ({
+  updatePortalConfiguration: data => dispatch(updatePortalConfigurationEvent(data)),
 });
 
 SettingsAccessLinkManagement.propTypes = {
   enterpriseUUID: PropTypes.string.isRequired,
   isUniversalLinkEnabled: PropTypes.bool.isRequired,
-  dispatch: PropTypes.func.isRequired,
+  couponsData: PropTypes.shape({
+    results: PropTypes.array,
```

Comment:
nit: I believe `PropTypes.array` is usually flagged by eslint, in favor of using `PropTypes.arrayOf(...)` here instead.